VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts

```diff
@@ -59,14 +60,30 @@ import { Room } from '../../../shared/models/room';
 import { RoomTransfer } from '../../../shared/models/room-transfer';
 import { ElementRef } from '@angular/core';
 import { VhToastComponent } from 'src/app/shared/toast/vh-toast.component';
-import { ConsultationInvitation } from '../../services/consultation-invitation.service';
+import { ConsultationInvitation, ConsultationInvitationService } from '../../services/consultation-invitation.service';
 import { Participant } from 'src/app/shared/models/participant';
+import { CallSetup } from '../../models/video-call-models';
```

Comment:
looks like call setup is actually not used in this test file. There are quite a lot of unused imports/variables in this file it seems, might be worth removing them?